main.c

```diff
@@ -320,7 +383,7 @@ void pressure_measurement_handler(void)
 			else
 			{
 				// of tep pressure
-				KalmanFiler1d_update(&vkf, tep_sensor.p/100, 0.25, 0.05);
+				KalmanFiler1d_update(&vkf, tep_sensor.p/100, 0.25, DELTA_TIME(tep_sensor.curtime,tep_sensor.prevtime));
```

Comment:
This change make sense, but please put this in a seperate commit as it nothing to do with temperature sensor support.
It is very difficult to track errors and do testing if different logic function are merged with one commit.